### PR TITLE
fix(lb): error check

### PR DESCRIPTION
**Pull request by Monitob**
* handle check 404 and 403 error on creation / read

### CODE REVIEW

File: scaleway/resource_lb.go [hunk 1]
```diff
@@ -153,7 +153,7 @@ func resourceScalewayLbCreate(ctx context.Context, d *schema.ResourceData, meta
 		RetryInterval: &retryInterval,
 	}, scw.WithContext(ctx))
 	// check err waiting process
-	if err != nil && !is404Error(err) {
+	if err != nil {
```
Comment (remyleone) [inline scaleway/resource_lb.go:156]: Why is this change required? When can we have a 404 in this case that is wanted?

Comment (Monitob) [reply scaleway/resource_lb.go:156]: On creation process, we should wait an stable state and confirm that the resource was created.

**Review by remyleone — APPROVED** [review]
